SSI.php

```diff
@@ -35,7 +35,7 @@
 if ((empty($cachedir) || !file_exists($cachedir)) && file_exists($boarddir . '/cache'))
 	$cachedir = $boarddir . '/cache';
 
-$ssi_error_reporting = error_reporting(defined('E_STRICT') ? E_ALL | E_STRICT : E_ALL);
+$ssi_error_reporting = error_reporting(E_ALL);
```

Comment:
What is the impact of this?

Reply:
E_STRICT (report on strict errors) didn't exist prior to 5.0 and prior to 5.4.0 asking for all errors would get you all error except strict ones. In 7, E_ALL includes strict mode too, so we don't explicitly need to ask for it.

Reply:
Nevermind, I see the commit now